src/launchers/cgi.ts

```diff
@@ -128,7 +129,7 @@ function query({ filename, path, host, headers, method, body }: PhpInput): Promi
     // PHP script execution end
     php.on('close', function (code, signal) {
       if (code !== 0) {
-        console.log(`🐘 PHP process closed code ${code} and signal ${signal}`);
+        debug(`🐘 PHP process closed code ${code} and signal ${signal}`);
```

Comment:
Same as above: is this necessary or a debug message (a.k.a. is it optional for the user?)